package.json

```diff
@@ -42,8 +41,8 @@
     "react-toastify": "^5.3.1",
     "redux": "^4.0.1",
     "redux-devtools-extension": "^2.13.8",
-    "redux-mock-store": "^1.5.3",
```

Comment:
@Manorlds-Eaglespark, what is the reason for changing this.

Reply:
This file was touched while trying out different ways of testing. It was added back when an understanding of how it works was established.